### ISSUE TITLE
Add Siteimprove script to the Storybook

### ISSUE THREAD

**Issue by vnys** [equinor/design-system#2701]
`<script async src="https://siteimproveanalytics.com/js/siteanalyze_6003171.js"></script>`

**Comment by BirteThornquist** [comment]
Need more information and clarification. Do we have licenses to use Siteimprove?

**Comment by oddvernes** [comment]
I have added the siteimprove link to the header of storybook when it is not in devlopment enviroment. It will turn up whenever the storybook cache refreshes (the clear cache step in actions does not work at the moment)

**Comment by oddvernes** [comment]
Cache is now refreshed but ublock-origin blocks it. It loads when I disable ublock-origin on the site. Are there any developers that doesn't have ublock installed? in bouvet it is force-installed by admin on all browsers 🤷

**Comment by vnys** [comment]
@vnys & @BirteThornquist will take a look at it after the standup

**Comment by nilsml** [comment]
We have data for eds.equinor.com and the storybook in SIteimprove it seems



![Image](https://github.com/equinor/design-system/assets/1836295/e910dbf1-cc91-4893-b28e-28c7de8beb74)

Who will be using this information?

**Comment by nilsml** [comment]
Looks like it is the dev version of storybook that shows up for some reason. Not the prod one.

**Comment by schoening** [comment]
We need to have a discussion about the need for siteimprove @vnys
